ember/app/templates/components/development-form.hbs

```diff
@@ -11,32 +11,56 @@
     {{/link-to}}
   {{/if}}
 
-  {{#if (and fulfilled changes)}}
-    {{#loading-spinner message=loadingSpinnerText isLoading=isLoading}}
-      <button onclick={{action update}} class='submit-changes styled'>{{submitText}}</button>
-    {{/loading-spinner}}
-  {{/if}}
+  {{#loading-spinner message=loadingSpinnerText isLoading=isLoading}}
+    <button onclick={{action update}} class='submit-changes styled' disabled={{not (and fulfilled changes)}}>
+      {{submitText}}
+    </button>
+  {{/loading-spinner}}
 </div>
 
-<div class="development-content">
+<form class="development-content">
   <div class="layout-column">
     <div class="development-header">
-      <h1 class="required">{{input value=editing.name placeholder="Development Name" key-up=(action checkForUpdated 'name')}}</h1>
-      <h2 class="required address-fields">
-        <img src="/assets/images/map-pin.svg" onclick={{action findPosition}}> 
-        {{input value=editing.address placeholder="Development Street Address" key-up=(action checkForUpdated 'address')}}
-        {{input value=editing.zipCode placeholder="Zip Code" key-up=(action checkForUpdated 'zipCode')}}
-      </h2>
-
-      <h2 class="project-url">
+
+      <div class="form-row required primary">
+        <label for="development_name" class="hidden">Development Name</label>
+        {{input
+          id="development_name"
+          value=editing.name
+          placeholder="Development Name"
+          key-up=(action checkForUpdated 'name')
+        }}
+      </div>
+
+      <div class="form-row required secondary location-fields">
+        <img src="/assets/images/map-pin.svg">
+        <label for="development_latitude" class="hidden">Latitude</label>
+        {{input
+          id="development_latitude"
+          value=(to-fixed editing.latitude 6)
+          placeholder="Latitude"
+          readonly="readonly"
+        }}
+        <label for="development_longitude" class="hidden">Longitude</label>
+        {{input
+          id="development_longitude"
```

Comment:
Can we remove the id?

Reply:
The `id` and `for` attributes are required to link the label and input for accessibility. Ideally this pattern would extend across the whole form, but that would have required more refactoring.